ecl/eclcc/eclcc.cpp

```diff
@@ -2157,9 +2190,9 @@ bool EclCC::processFiles()
     }
     else if (inputFiles.ordinality() == 0)
     {
-        assertex(optQueryRepositoryReference);
+        assertex(optQueryMainAttribute);
```

Comment:
This assertex is not necessary as the tests conditions & logic in lines 2144-2150 means that this cannot ever be false.